### PR TITLE
fix: Fix and refactor digitization parametrization in Examples

### PR DESCRIPTION
**Pull request by andiwand**
- `Exact` was not serialized correctly
- Add sigma to `Exact` as `0` breaks reconstruction
- Configure `GaussTrunc` and `GaussClipped` range in terms of `sigma`

### CODE REVIEW

File: Examples/Io/Json/src/JsonDigitizationConfig.cpp [hunk 1]
```diff
@@ -69,9 +69,10 @@ void to_json(nlohmann::json& j, const ActsFatras::SingleParameterSmearFunction<
     return;
   }
   // Exact
-  auto exact = f.target<const Digitization::Digital>();
```
Comment (benjaminhuth) [inline Examples/Io/Json/src/JsonDigitizationConfig.cpp:72]: Looks like I was lazy and did copy-paste here^^

**Review by benjaminhuth — APPROVED** [review]
Looks good!

File: Examples/Algorithms/Digitization/include/ActsExamples/Digitization/Smearers.hpp [hunk 2]
```diff
@@ -26,6 +26,11 @@ namespace ActsExamples::Digitization {
 /// Exact smearing of a single parameter.
 ///
 struct Exact {
+  double sigma;
+
+  /// Construct with a @param sigma standard deviation
+  Exact(double sigma_) : sigma{sigma_} {}
```
Comment (paulgessinger) [inline Examples/Algorithms/Digitization/include/ActsExamples/Digitization/Smearers.hpp:32]:
```suggestion
  explicit Exact(double sigma_) : sigma{sigma_} {}
```

Maybe we should do this for the other ones as well.

**Review by paulgessinger — APPROVED** [review]

**Review by benjaminhuth — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results [b4c888d828d70cb003f24efcca596cd18569c244]
### Build job with this PR failed!
Please investigate the [build job](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/41257454) for the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/7760684)!

**Pull request merged**